selenium_tests/course_info/course_info_is_initialized_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from selenium_tests.course_info.course_info_base_test_case import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.course_info_parent_page_object import CourseInfoParentPage
+
+
+class CourseInfoIsInitializedTest(CourseInfoBaseTestCase):
+
+    def test_is_page_loaded(self):
+
+        """
+        Check if page is loaded
+        """
+        driver = self.driver
+        # initialize
+        page = CourseInfoParentPage(driver)
+        page.is_loaded()
```

Comment:
No assert in this test?.... Also the test name is a bit confusing -- this appears to check whether the parent page is loaded, but that isn't clear to me from the class name + method name combination.